src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -172,72 +213,99 @@ def annotate(
+            values will be list of VRS data. Else, will be an empty dictionary
         :param str assembly: The assembly used in `vcf_coords`
         :param Optional[str] vrs_data_key: The key to update in `vrs_data`. If not
             provided, will use `vcf_coords` as the key.
         :param bool output_pickle: `True` if VRS pickle file will be output.
             `False` otherwise.
         :param bool output_vcf: `True` if annotated VCF file will be output.
             `False` otherwise.
+        :param bool vrs_attributes: If `True` will include VRS_Start, VRS_End,
+            VRS_Alt fields in the INFO field. If `False` will not include these fields.
+            Only used if `output_vcf` set to `True`.
         """
         try:
             vrs_obj = self.tlr._from_gnomad(vcf_coords, assembly_name=assembly)
         except (ValidationError, TranslatorValidationError) as e:
+            vrs_obj = None
             _logger.error("ValidationError when translating %s from gnomad: %s", vcf_coords, str(e))
         except KeyError as e:
+            vrs_obj = None
             _logger.error("KeyError when translating %s from gnomad: %s", vcf_coords, str(e))
         except AssertionError as e:
+            vrs_obj = None
             _logger.error("AssertionError when translating %s from gnomad: %s", vcf_coords, str(e))
         except Exception as e:  # pylint: disable=broad-except
+            vrs_obj = None
             _logger.error("Unhandled Exception when translating %s from gnomad: %s", vcf_coords, str(e))
         else:
-            if vrs_obj:
-                if output_pickle:
-                    key = vrs_data_key if vrs_data_key else vcf_coords
-                    vrs_data[key] = str(vrs_obj.as_dict())
-
-                if output_vcf:
-                    vrs_allele_ids.append(vrs_obj._id._value)
-            else:
+            if not vrs_obj:
                 _logger.debug("None was returned when translating %s from gnomad", vcf_coords)
 
-    def _record_digests(  # pylint: disable=too-many-arguments
+        if output_pickle and vrs_obj:
+            key = vrs_data_key if vrs_data_key else vcf_coords
+            vrs_data[key] = str(vrs_obj.as_dict())
+
+        if output_vcf:
+            allele_id = vrs_obj._id._value if vrs_obj else ""
+            vrs_field_data[self.VRS_ALLELE_FIELD].append(allele_id)
+
+            if vrs_attributes:
+                start = str(vrs_obj.location.interval.start.value) if vrs_obj else ""
+                end = str(vrs_obj.location.interval.end.value) if vrs_obj else ""
+                alt = str(vrs_obj.state.sequence) if vrs_obj else ""
+
+                vrs_field_data[self.VRS_START_FIELD].append(start)
+                vrs_field_data[self.VRS_END_FIELD].append(end)
+                vrs_field_data[self.VRS_ALT_FIELD].append(alt)
+
+    def _record_digests(  # pylint: disable=too-many-arguments,too-many-locals
```

Comment:
we should update this method name, maybe _get_vrs_data()

Reply:
Updated